backfiller/backfiller/main.py

```diff
@@ -171,7 +174,7 @@ def start_worker(self, node):
 		if self.stopping.is_set():
 			logging.debug('Refusing to create new workers because we are stopping')
 			return
-		worker = BackfillerWorker(self, node)
+		worker = BackfillerWorker(self, node, self.download_concurrency)
```

Comment:
Minor point but the the workers get their other parameters from the manager rather than from the constructor arguments.

Reply:
fixed